src/cdk/a11y/aria-reference.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+/** IDs are deliminated by an empty space, as per the spec. */
+const ID_DELIMINATOR = ' ';
```

Comment:
We should probably be tolerant of any amount of whitespace between IDs rather than a single space (even though the spec calls for a single space)